static/src/javascripts/bootstraps/enhanced/newsletters.js

```diff
@@ -125,6 +135,7 @@ const enhanceNewsletters = (): void => {
         getUserFromApi(userFromId => {
             if (userFromId && userFromId.primaryEmailAddress) {
                 updatePageForLoggedIn(userFromId.primaryEmailAddress);
+                $.forEachElement(`.${classes.signupButton}`, modifyForSignedIn);
```

Comment:
I wonder whether somewhere we should be utilising fastdom.read to queue up the DOM reads

Reply:
Similar to one of my other comments - I'd rename `subscribeToEmail` (not a good name anyway IMO) - and then include modification of the link name as part of the same place that is called. Maybe something like:

```
$.forEachElement(`.${classes.signupButton}`, modifyFormForSignedIn);

const modifyFormForSignedIn = (el) => {
    modifyLinkNamesForSignedIn(el);
    createFormEventHandlers(el);
}```